.circleci/config.yml

```diff
@@ -188,7 +188,7 @@ jobs:
           command: |
             RAILS_ENV=test bundle exec ruby ./bin/coverage.rb
       - store_artifacts:
-          path: ~/figgy/coverage
```

Comment:
![](https://media.giphy.com/media/3wPQiJa2tAB8c/giphy.gif)

Reply:
Ugh. I updated that in a previous commit. Must have removed it somehow.

Reply:
![figgy](https://user-images.githubusercontent.com/784196/89669567-43fbcb80-d8a5-11ea-8579-6a77cf041186.jpg)

Reply:
![](https://media.giphy.com/media/pXJksds5tRAic/giphy-downsized.gif)